### PR TITLE
Adapt CI to match version generation between build and qemu-bundles-tests steps

### PR DESCRIPTION
**Pull request by mauromorales**
Signed-off-by: Mauro Morales <mauro.morales@spectrocloud.com>

**What this PR does / why we need it**:

The test with kubo was failing because there's a version check done by `systemd-sysext` and it didn't match. The issue comes from the new way of fetching the version from git, which in the build step we use the full history and tags so it will look something like `v1.2.3-sha-dirty` while in the `qemu-bundles-tests` we fetch only the last commit without tags or history so it looks just like `sha`. 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

Let's make master green again!

### CODE REVIEW

File: .github/workflows/image.yaml [hunk 1]
```diff
@@ -189,6 +189,8 @@ jobs:
          - flavor: "opensuse"
     steps:
     - uses: actions/checkout@v3
+      - run: |
```
Comment (mudler) [inline .github/workflows/image.yaml:192]: what about the release pipeline?

Comment (mauromorales) [reply .github/workflows/image.yaml:192]: @mudler for what I can tell both release.yaml and release-arm.yaml are ok, but I'm not entirely sure about release_bin.yaml does goreleaser somehow does any building too? if not then I think we're good

File: .github/workflows/image.yaml [hunk 2]
```diff
@@ -34,21 +34,21 @@ jobs:
...
+      - name: login to quay registry
         if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
-        run: echo ${{ secrets.QUAY_PASSWORD }} | docker login -u ${{ secrets.QUAY_USERNAME }} --password-stdin quay.io
-      - name: Build  🔧
+        run: echo ${{ secrets.quay_password }} | docker login -u ${{ secrets.quay_username }} --password-stdin quay.io
+      - name: build  🔧
         env:
-          FLAVOR: ${{ matrix.flavor }}
-          IMAGE: quay.io/kairos/core-${{ matrix.flavor }}:latest
+          flavor: ${{ matrix.flavor }}
```
[the remark this answers is not in the document]
Comment (mauromorales) [reply .github/workflows/image.yaml:48]: :facepalm: ah crap ... I blame it on Vim :P ... fixing

**Comment by mudler** [comment]
awesome, good catch!

**Review by mudler — APPROVED** [review]

**Pull request merged**